### PR TITLE
feat: add scripts to sync PIDs and Qualities on external servers

### PR DESCRIPTION
**Pull request by dtroelofsprins**
#### Checklist
- [x] Functionality works & meets specifications
- [ ] Code reviewed
- [x] Code unit/system tested
- [ ] User documentation updated
- [ ] Clean commits
- [ ] Updated CHANGELOG.md

### CODE REVIEW

File: scripts/sync_ERIC_PIDs.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        try:
+            response.raise_for_status()
+        except requests.RequestException as ex:
+            raise_exception(ex)
+
+        return response
+
+
+session = ExtendedSession(url="http://localhost:8080", token="${molgenisToken}")
+eric_session = Session(url="https://directory.bbmri-eric.eu")
```
Comment (jelmerveen) [inline scripts/sync_ERIC_PIDs.py:44]: hardcoded url to production?

File: scripts/sync_ERIC_qualities.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+"""
+
+from molgenis.client import Session
+
+tables = {
+    "eu_bbmri_eric_biobanks": ["eu_bbmri_eric_bio_qual_info", "biobank"],
+    "eu_bbmri_eric_collections": ["eu_bbmri_eric_col_qual_info", "collection"],
+}
+session = Session(url="http://localhost:8080", token="${molgenisToken}")
+eric_session = Session(url="https://directory.bbmri-eric.eu")
```
Comment (jelmerveen) [inline scripts/sync_ERIC_qualities.py:16]: same here

Comment (jelmerveen) [reply scripts/sync_ERIC_qualities.py:16]: maybe use a variable instead.

**Review by jelmerveen — CHANGES_REQUESTED** [review]

File: scripts/sync_ERIC_qualities.py (hunk 2 above)
Comment (dtroelofsprins) [reply scripts/sync_ERIC_qualities.py]: You are totally right, replaced the hard-coded URL in both scripts by a parameter

File: scripts/sync_ERIC_PIDs.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+from molgenis.errors import raise_exception
+
+
+class ExtendedSession(Session):
+    """
+    Class adding batch update of one attribute to the molgenis-py-client
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(ExtendedSession, self).__init__(*args, **kwargs)
```
Comment (jelmerveen) [inline scripts/sync_ERIC_PIDs.py:25]: Just curious: Why do you need to pass the class as an argument for super?

Comment (dtroelofsprins) [reply scripts/sync_ERIC_PIDs.py:25]: To be honest: it's a copy paste from other code (developed by Tommy). Did a bit of reading and you are right, there is no need for it, it's equivalent to the parameterless super() call.

File: scripts/sync_ERIC_qualities.py (hunk 2 above)
Comment (dtroelofsprins) [reply scripts/sync_ERIC_qualities.py]: Remove the subclass argument

File: scripts/sync_ERIC_PIDs.py (hunk 3 above)
Comment (dtroelofsprins) [reply scripts/sync_ERIC_PIDs.py]: Removed the subclass argument

**Review by jelmerveen — APPROVED** [review]

**Pull request merged**